mmdet3d/core/evaluation/kitti_utils/eval.py

```diff
@@ -473,8 +473,9 @@ def eval_class(gt_annos,
     assert len(gt_annos) == len(dt_annos)
     num_examples = len(gt_annos)
     if num_examples < num_parts:
-        num_parts = num_examples
+        num_parts = 1
```

Comment:
Is this change necessary?

Reply:
The original implementation may trigger error while `num_examples` smaller than `num_parts`